packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx

```diff
@@ -225,6 +273,7 @@ export class FbFieldDropdownItem {
       strDragClassName += ` ${strBaseDragClassName}--unsortable`;
     }
 
+    const numNameMaxChars = objMaxLimits?.['maxChoiceChars']?.count || 255;
```

Comment:
@Srikannan-E You can directly access count here instead of `objMaxLimits?.['maxChoiceChars']?.count`. Can you please update and test this one?

Reply:
All changes done bro